kafka-adapter/kafka-adapter.go

```diff
@@ -0,0 +1,120 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/Shopify/sarama"
+	"github.com/juju/errors"
+	"github.com/ngaut/log"
+	"github.com/unrolled/render"
+	"net/http"
+	"os"
+	"os/signal"
+	"sync"
+	"syscall"
+	"time"
+)
+
+var (
+	port         = flag.Int("port", 28082, "http listen port")
+	kafkaAddress = flag.String("kafka-address", "10.0.3.4:9092,10.0.3.5:9092,10.0.3.6:9092", "kafka adddress")
+	kafkaTopic   = flag.String("kafka-topic", "test", "kafka topic")
+	logFile      = flag.String("log-file", "kafka-adapter.log", "log file")
+	logLevel     = flag.String("log-level", "info", "log level: debug, info, warn, error, fatal")
+)
+
+//KafkaMsg represent kafka msg
+type KafkaMsg struct {
+	Title       string `json:"title"`
+	Source      string `json:"source"`
+	Node        string `json:"node"`
+	Expr        string `json:"expr"`
+	Description string `json:"description"`
+	URL         string `json:"url"`
+	Level       string `json:"level"`
+	Note        string `json:"note"`
+	Value       string `json:"value"`
+	Time        string `json:"time"`
+}
+
+//Run represent runtime informations
+type Run struct {
+	Rdr         *render.Render
```

Comment:
struct Run defined in kafka-adapter.go and interface also in kafka.go, uniform it